### ISSUE TITLE
Support Gnome 40

### ISSUE THREAD

**Issue by Lucki** [timbertson/gnome-shell-scroll-workspaces#43]
With Gnome 40 the workspace layout changed to horizontal and this extension is marked as unsupported.
Please update the extension as the behavior could be the same.

The **mouse** shortcut set by Gnome to change the workspaces is "holding Super (on the keyboard :facepalm:) + scroll wheel" which doesn't work most of the time because the currently hovered application scrolls instead.

Manually changing the supported version to 40 seems to pick up the extension properly but it doesn't do anything.
Trying to open the extension settings yields:

~~~
Error: No property border_width on GtkBox

Stack trace:
  _init/Gtk.Widget.prototype._init@resource:///org/gnome/gjs/modules/core/overrides/Gtk.js:45:40
  buildPrefsWidget@/home/lucki/.local/share/gnome-shell/extensions/scroll-workspaces@gfxmonk.net/prefs.js:15:14
  _init@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:209:40
  OpenExtensionPrefsAsync/<@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:122:28
  asyncCallback@resource:///org/gnome/gjs/modules/core/overrides/Gio.js:115:22
  run@resource:///org/gnome/Shell/Extensions/js/dbusService.js:177:20
  main@resource:///org/gnome/Shell/Extensions/js/main.js:19:13
  run@resource:///org/gnome/gjs/modules/script/package.js:206:19
  start@resource:///org/gnome/gjs/modules/script/package.js:190:8
  @/usr/share/gnome-shell/org.gnome.Shell.Extensions:1:17
~~~

Please let me know if you need more information and how I can get them.

**Comment by xuhcc** [comment]
Top panel workspace scroll still works if you use vertical overview extension https://github.com/RensAlthuis/vertical-overview

**Comment by Lucki** [comment]
That's great to hear although I'm not completely against vertical workspaces, more against the bad side affects e.g. the long mouse move to your favorites or the really tiny workspace previews. I'd like to have less extensions that can break the next gnome version but I'll check it out.

There's also [this extension](https://github.com/sunwxg/gnome-shell-extension-panelScroll) that got updated but there you have to remind you all the time to only use the right panel side…

**Comment by Lucki** [comment]
Solved by #44
